inventory/src/main/java/org/opennms/horizon/inventory/model/MonitoringLocations.java

```diff
@@ -0,0 +1,55 @@
+package org.opennms.horizon.inventory.model;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.validation.constraints.NotNull;
+import java.util.UUID;
+
+// TODO: Lombok isn't playing nicely with MapStruct, so generate getters and setters with IDE for now.
+//@Getter
+//@Setter
+@RequiredArgsConstructor
+@Entity
+public class MonitoringLocations {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @NotNull
+    @Type(type="pg-uuid")
+    private UUID tenant_id;
```

Comment:
The following change will make the mapper simpler
```suggestion
@Column(name = "tenant_id")
private UUID tenantId;
```

Reply:
Will change